packages/vue-app/package.json

```diff
@@ -5,9 +5,12 @@
   "license": "MIT",
   "files": [
     "dist",
-    "template"
+    "template",
+    "types/index.d.ts",
```

Comment:
Listing `types` would be enough (As by-convention we use a separate directory for `tests`.

Reply:
Oh alright, I've just understood that `vue`, `vue-router`, etc... were listing each file to not include `tests` in the final published NPM package.

Indeed, listing `types` would be enough !

I will update this at the same time as the `tests`, but where should be the `test`/`tests` folder then ?

Reply:
Currently, test fixtures are inside root `test/fixtures` so I suggest creating testable ts project for linting inside `test/fixtures/typescript`. I know this is not the best place but there are plans for moving each package tests into its own.

Reply:
Alright, I'll dive into it soon (this week). If I'm facing any problems I'll post them in the main PR thread.

Reply:
Maybe could we let tests inside `vue-app` and in root `package.json` have the script `test:types` to be `tsc -p packages/vue-app/types/test`

Reply:
Tests really need to be "close" to the typings tested

Reply:
@kevinmarrec As I said that's handled by Lerna. Types are also symlinked into `[root]/node_modules/@nuxt/vue-app/types/index`. And listing `@nuxt/vue-app` in tsconfig, _should_ be enough. If you encounter problems just commit and ping me :)

Reply:
@pi0 Lerna handle this gently when running `tsc`, for example to test `Transition type`

```ts
import { Transition } from '@nuxt/vue-app'

// Pass
const test1: Transition = {
  name: 'pass'
}

// Fail
const test2: Transition = {
  fail: 'fail'
}
```

BUT it's a pain to code de tests, I only have autocompletion within editor if I do 
```ts
import { Transition } from '../../packages/vue-app'
```